### PR TITLE
Managing reservation

### PR DESCRIPTION
**Pull request by clooux**

### CODE REVIEW

**Comment by Denwiliusz27** [comment]
got an error trying to open room view
image

**Comment by Denwiliusz27** [comment]
while adding new reservation, it saves on wrong hours
image
image

File: frontend/src/utils/ChangeClassType.ts [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
-    } else if (classType["pl"]=="Warsztat"){
-        classType["pl"] = "WAR"
-    }
-    return classType;
-}
\ No newline at end of file
+export function changeClasstypeName({
+  classType,
+}: {
+  classType: Map<string, string>;
```
Comment (Denwiliusz27) [inline frontend/src/utils/ChangeClassType.ts:4]: got error here
image

**Comment by bestemic** [comment]
Updating reservation info is missing too.

**Comment by Denwiliusz27** [comment]
different hours on modification popup
image

**Comment by Denwiliusz27** [comment]
the clock doesn't display correctly
image

**Comment by Denwiliusz27** [comment]
calendar same
image

**Comment by Denwiliusz27** [comment]
after clicking "modyfikuj" and then "odrzuć", go back to reservations view and again on reservation, it says that it is "Zaakceptowana"

File: frontend/src/pages/Reservation.tsx [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+          headers: {
+            Authorization: `Bearer ${token}`,
+          },
+        }
+      );
+      return response.data;
+    },
+    onSuccess: (data) => {
+      reservationQuery.refetch();
+      toast.success("Rezerwacja została odrzucona");
```
Comment (Denwiliusz27) [inline frontend/src/pages/Reservation.tsx:120]: toast hide too quickly

File: frontend/src/pages/Reservation.tsx [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+          headers: {
+            Authorization: `Bearer ${token}`,
+          },
+        }
+      );
+      return response.data;
+    },
+    onSuccess: (data) => {
+      reservationQuery.refetch();
+      toast.success("Rezerwacja została potwierdzona!");
```
Comment (Denwiliusz27) [inline frontend/src/pages/Reservation.tsx:139]: same here

**Comment by Denwiliusz27** [comment]
reservations adds on previously set reservation

**Review by Denwiliusz27 — APPROVED** [review]
lgtm

**Pull request merged**